src/main/kotlin/no/nav/familie/baks/mottak/integrasjoner/BaSakClient.kt

```diff
@@ -172,6 +172,8 @@ data class RestMinimalFagsak(
 class RestVisningBehandling(
     val behandlingId: Long,
     val opprettetTidspunkt: LocalDateTime,
+    val kategori: BehandlingKategori,
+    val underkategori: BehandlingUnderkategori?,
```

Comment:
Denne er null for KS

Reply:
Kan dokumentere dette gjennom typer? F.eks:

sealed class RestVisningBehandling(
    val behandlingId: Long,
    val opprettetTidspunkt: LocalDateTime,
    val kategori: BehandlingKategori,
    val aktiv: Boolean,
    val årsak: String? = null,
    val type: String,
    val status: String,
    val resultat: String? = null,
    val vedtaksdato: LocalDateTime? = null
) {
    data class Ba(
        val underkategori: BehandlingUnderkategori
    ) : RestVisningBehandling(
        behandlingId,
        opprettetTidspunkt,
        kategori,
        aktiv,
        årsak,
        type,
        status,
        resultat,
        vedtaksdato
    )

    data class Ks : RestVisningBehandling(
        behandlingId,
        opprettetTidspunkt,
        kategori,
        aktiv,
        årsak,
        type,
        status,
        resultat,
        vedtaksdato
    )
}